Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -455,6 +455,18 @@ struct ImageMetadata {
     }
 };
 
+static ErrorOr<void> ensure_metadata_correctness(ImageMetadata const& metadata)
```

Comment:
very nit: I would've written the commit message as "Disallow CMYK images without ICC profile

CMKY images are detected by the presence of a kBlack channel."